tests/base/src/test/java/org/keycloak/tests/model/InfinispanTimeUtil.java

```diff
@@ -0,0 +1,40 @@
+package org.keycloak.tests.model;
```

Comment:
I propose to move it to utils module (tests/utils) with a specific package name: org/keycloak/tests/utils/model.